virtusize/src/main/java/com/virtusize/android/ui/VirtusizeWebViewFragment.kt

```diff
@@ -178,6 +179,11 @@ class VirtusizeWebViewFragment : DialogFragment() {
         binding.webView.loadUrl(virtusizeWebAppUrl)
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        VirtusizeAuth.handleVirtusizeAuthResult(binding.webView, requestCode, resultCode, data)
```

Comment:
Please refer to https://github.com/virtusize/virtusize-auth-android/tree/feature/facebook-auth